extension/popup/popupView.jsx

```diff
@@ -589,23 +595,50 @@ const SearchResultsContent = ({
     onNextSearchResultClick();
   };
 
+  const searchCard = () => (
```

Comment:
We've been capitalizing these functions that return markup, so SearchCard would be a better name.

Reply:
I will make this update!